aws-connect

```diff
@@ -19,9 +19,12 @@ version() {
 usage() {
   echo "Usage:"
   echo ""
-  echo "$programname -a ssh|tunnel [-n <instance name>|-t <instance tag>] [-r region] [-p profile name] [-o port] [-x instance id] [-s] [-h] [-v]"
+  echo "$programname -a ssh|tunnel|document [-d document name] [-c document parameters] [-g github access token location] [-n <instance name>|-t <instance tag>] [-r region] [-p profile name] [-o port] [-x instance id] [-s] [-h] [-v]"
```

Comment:
Consistency (which is my bad).  Google for the "standard" for putting args in the help and make them standard across all of these (ie. should we have `<>` or not?